src/builder/components/question/card/survey-question-card.component.tsx

```diff
@@ -0,0 +1,98 @@
+import { message } from 'antd';
+import React, { Component } from 'react';
+import { SurveyItem } from '../../../../common/types/survey-item.type';
+import { EditableTitle } from '../editable-title/survey-question-editable-title.component';
+import { SurveyQuestionHighlight } from '../highlight/survey-question-highlight.component';
+import { SurveyQuestion } from '../survey-questions/survey-question.component';
+import { SurveyQuestionToolbox, ToolboxActions } from '../toolbox/survey-question-toolbox.component';
+import './survey-question-card.component.less';
+
+export interface SurveyQuestionCardProps {
+    question: SurveyItem;
+    onDelete: (itemId: string) => void;
+    onUpdate: (item: SurveyItem) => void;
+}
+
+export interface SurveyQuestionCardState {
+    isToolboxVisible: boolean;
+}
+
+export class SurveyQuestionCard extends Component<SurveyQuestionCardProps, SurveyQuestionCardState> {
+    state = {
+        isToolboxVisible: false,
+    };
+
+    deleteMessage = () => {
+        message.success(`${this.props.question.description} removed successfully!`);
+    };
+
+    showToolbox = () => this.setState({ isToolboxVisible: true });
+    hideToolbox = () => this.setState({ isToolboxVisible: false });
+
+    editSurveyTitle = (newTitle: string) => {
+        const { question } = this.props;
+        question.description = newTitle;
+        this.props.onUpdate(question);
+    };
+
+    render() {
+        const { question } = this.props;
+        const { type } = this.props.question;
```

Comment:
Why don't you use the `question` const that you just assigned?

Reply:
Must have gotten lost somewhere in the refactoring. I'll work on getting it consistent because it seems like a lot of these creeped in.

I don't usually unbox props this way, which is why it slipped through.